verify_color.py

```diff
@@ -0,0 +1,16 @@
+from classification_colours import MAJOR_COLORS, MINOR_COLORS
+
+def get_color_from_pair_number(pair_number):
+  zero_based_pair_number = pair_number - 1
+  major_index = zero_based_pair_number // len(MINOR_COLORS)
+  if major_index >= len(MAJOR_COLORS):
+    raise Exception('Major index out of range')
+  minor_index = zero_based_pair_number % len(MINOR_COLORS)
+  if minor_index >= len(MINOR_COLORS):
+    raise Exception('Minor index out of range')
+  return MAJOR_COLORS[major_index], MINOR_COLORS[minor_index]
+
+def verify_color(expected_major_color, expected_minor_color, pair_number):
```

Comment:
Test code and production code are both part of single file. It is a good idea to keep them separate so that maintenance is easy and production code can be tested independently and effectively.